extensions/functions_string.yaml

```diff
@@ -88,7 +88,11 @@ scalar_functions:
       from the beginning of the string to begin starting to search for pattern matches can be
       specified using the `position` argument. Specifying `1` means to search for matches
       starting at the first character of the input string, `2` means the second character, and so
-      on. The `position` argument should be a positive non-zero integer.
+      on. The `position` argument should be a positive non-zero integer. The regular
+      expression capture group can be specified using the `group` argument. Specifying `0`
```

Comment:
I'm confused (never surprising :smile:  ).  What is the difference is between `group` and `occurrence`?

Reply:
Ok.  I think I see now.  If I match the string `(0-50),(0-70)` with the pattern `(\d+)-(\d+)` then there are two occurrences `0-50` and `0-70` and two capture groups (the first matches `0` and `0` and the second matches `50` and `70`).